requirements.txt

```diff
@@ -1,3 +1,4 @@
 flexmock==0.9.7
 py==1.4.13
 pytest==2.3.4
+tox==2.3.1
```

Comment:
Actually, this whole file can be removed since these dependencies are specified in `tox.ini`.